test/tests/core/Mat/imgprocTests.js

```diff
@@ -1479,5 +1479,64 @@ module.exports = (getTestImg) => {
         expectOutput: makeExpectOutput(cv32fc6)
       });
     });
+
+    describe('goodFeaturesToTrack', () => {
+      generateAPITests({
+        getDut,
+        methodName: 'goodFeaturesToTrack',
+        methodNameSpace: 'Mat',
+        getRequiredArgs: () => ([
+          20, 0.04, 1
+        ]),
+        getOptionalArgs: () => ([
+          new cv.Mat(512,512,cv.CV_U8), // mask
+          3, // blockSize
+          3, // gradientSize
+          false, // useHarrisDetector
+          0.04 // harrisK
+        ]),
```

Comment:
The `getOptionalArgs` can be removed. Optional args are already tested when `getOptionalArgsMap` is given. We only use `getOptionalArgs` if the function does not provide the option to take a JSON object with named parameters.